build_system/src/build.rs

```diff
@@ -171,8 +173,9 @@ pub fn build_sysroot(env: &HashMap<String, String>, config: &ConfigInfo) -> Resu
     Ok(())
 }
 
+/// Builds the codegen for the specified arguments.
```

Comment:
Please remove this comment as it is a duplicate of the code itself, so it doesn't bring anything more than the code.